### PR TITLE
Make node view is similar to cluster view

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Adjust Node view resources to cluster view resources
- Use ProgressIndicatorWithPercentage in Cluster view node list

**Related issue(s)**

### CODE REVIEW

File: src/components/Nodes/MachineInfo/MachineInfo.js [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
-        <LayoutPanelRow
-          name={t('machine-info.kube-proxy-version')}
-          value={nodeInfo.kubeProxyVersion}
-        />
-        <LayoutPanelRow
-          name={t('machine-info.kubelet-version')}
-          value={nodeInfo.kubeletVersion}
-        />
+    <>
+      <div className="cluster-overview__graphs-wrapper">
```
Comment (mrCherry97) [inline src/components/Nodes/MachineInfo/MachineInfo.js:15]: this div with className is not necessary probably, after removing this nothing changed.

File: src/components/Nodes/NodeDetails/NodeDetails.js [hunk 2]
```diff
@@ -23,7 +25,7 @@ function NodeDetails({ nodeName }) {
       defaultType={EVENT_MESSAGE_TYPE.WARNING}
     />
   );
-
+  console.log(data);
```
Comment (mrCherry97) [inline src/components/Nodes/NodeDetails/NodeDetails.js:28]: remove clg

File: src/components/Nodes/NodeDetails/NodeDetails.js [hunk 3]
```diff
@@ -10,6 +10,8 @@ import { EVENT_MESSAGE_TYPE } from 'hooks/useMessageList';
 import { spacing } from '@ui5/webcomponents-react-base';
 import './NodeDetails.scss';
 import YamlUploadDialog from 'resources/Namespaces/YamlUpload/YamlUploadDialog';
+import { Title } from '@ui5/webcomponents-react';
+import React from 'react';
```
Comment (mrCherry97) [inline src/components/Nodes/NodeDetails/NodeDetails.js:14]: this import is unused

File: src/components/Nodes/NodeResources/NodeResources.js [hunk 4]
```diff
@@ -1,44 +1,60 @@
...
-              position: 'bottom',
-            }}
-          />
-        </div>
-      ) : (
-        t('components.error-panel.error')
-      )}
-    </UI5Panel>
+    <>
+      <div className="cluster-overview__graphs-wrapper">
```
Comment (mrCherry97) [inline src/components/Nodes/NodeResources/NodeResources.js:12]: everything is "packed" in `div` so `<>` this react fragment is unnecessary.
```suggestion
      <div className="cluster-overview__graphs-wrapper">
```

File: src/components/Nodes/NodeResources/NodeResources.js [hunk 5]
```diff
@@ -1,44 +1,60 @@
...
+                  position: 'bottom',
+                }}
+              />
+            </Card>
+          </>
+        ) : (
+          t('components.error-panel.error')
+        )}
+      </div>
+    </>
```
Comment (mrCherry97) [inline src/components/Nodes/NodeResources/NodeResources.js:58]: As the previous comment.
```suggestion
      </div>
```

File: src/components/Nodes/NodeResources/NodeResources.js [hunk 6]
```diff
@@ -1,44 +1,60 @@
...
+          <>
+            <Card
+              header={
+                <CardHeader
+                  titleText={t('cluster-overview.statistics.cpu-usage')}
+                />
+              }
+            >
+              <UI5RadialChart
+                color="var(--sapIndicationColor_7)"
```
Comment (mrCherry97) [inline src/components/Nodes/NodeResources/NodeResources.js:23]: we need to talk about colors, idk if we want to have the same colors as on the cluster and namespace overview.

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: src/components/Clusters/views/ClusterOverview/ClusterNodes.js [hunk 7]
```diff
@@ -5,12 +5,12 @@ import { useTranslation } from 'react-i18next';
 import { EventsList } from 'shared/components/EventsList';
 import { EVENT_MESSAGE_TYPE } from 'hooks/useMessageList';
 import { GenericList } from 'shared/components/GenericList/GenericList';
-import { ProgressBar } from 'shared/components/ProgressBar/ProgressBar';
 import { ReadableCreationTimestamp } from 'shared/components/ReadableCreationTimestamp/ReadableCreationTimestamp';
 
 import { EMPTY_TEXT_PLACEHOLDER } from 'shared/constants';
 import { StatusBadge } from 'shared/components/StatusBadge/StatusBadge';
 import { useUrl } from 'hooks/useUrl';
+import { ProgressIndicatorWithPercentage } from '../../../../shared/components/ProgressIndicatorWithPercentage/ProgressIndicatorWithPercentage';
```
Comment (akucharska) [inline src/components/Clusters/views/ClusterOverview/ClusterNodes.js:13]:
```suggestion
import { ProgressIndicatorWithPercentage } from 'shared/components/ProgressIndicatorWithPercentage/ProgressIndicatorWithPercentage';
```

**Review by nataliasitko — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**